ui/packages/consul-ui/tests/acceptance/dc/services/show.feature

```diff
@@ -106,11 +106,11 @@ Feature: dc / services / show: Show Service
     And settings from yaml
     ---
     consul:urls:
-      service: https://consul.io?service-name={{Service.Name}}&dc={{Datacenter}}
+      service: https://example.com?service-name={{Service.Name}}&dc={{Datacenter}}
     ---
     When I visit the service page for yaml
     ---
       dc: dc1
       service: service-0
     ---
-    And I see href on the dashboardAnchor like "https://consul.io?service-name=service-0&dc=dc1"
+    And I see href on the dashboardAnchor like "https://example.com?service-0-with-id&dc1"
```

Comment:
I'll try making the ui-config configurable. I'll remove the input field out the settings page.

Reply:
Yeah something like:

```
And ui_config from yaml
---
dashboard_metrics_url:
  services: http://consul.io/
---
```

It would be great if we could use the same cookies thing we use so we can configure it during development aswell. Sounds like this is something we can do at a later date then?